Content.Shared/Fax/FaxVisuals.cs

```diff
@@ -13,5 +13,8 @@ public enum FaxMachineVisualState : byte
 {
     Normal,
     Inserting,
+    InsertingHamlet,
+    InsertingMouse,
+    InsertingMothroach,
     Printing
 }
```

Comment:
As above.

Reply:
Not really sure what to do here. surely the inserting states need to be listed here?

Reply:
As mentioned split it out so we don't have faxes requiring mobstate to exist.

The sprites will also need splitting out because they just copy the fax sprite over and make any new sprites a pain in the ass to add.